src/templates/admin/cron/reminders.html

```diff
@@ -43,8 +36,7 @@ <h2>Active Reminders</h2>
                         {% for reminder in reminders %}
                             <tr>
                                 <td>{{ reminder.pk }}</td>
-                                <td>Sent {{ reminder.days }} days {{ reminder.run_type }} {{ reminder.get_type_display }} is
-                                    due.</td>
+                                <td>Sent {{ reminder.days }} days {{ reminder.run_type }} {{ reminder.get_type_display }} is due.</td>
```

Comment:
I have a request about the way the choices and this sentence are formed, since I think it's key to the clarity problem.

I added some detail here:
https://github.com/BirkbeckCTP/janeway/issues/3427#issuecomment-1554220703

What if we make this line like this:
```html
<td>Sent {{ reminder.days }} days {{ reminder.run_type }} for {{ reminder.get_type_display }}.</td>
```

And change the choices to:
```py
RUN_TYPE_CHOICES = (
    # Before the event
    ('before', 'Before the due date'),
    # After the event
    ('after', 'After the due date'),
)
```

Reply:
I think this would be a good change! 👍🏻